### ISSUE TITLE
What does CC-LICENSE refer to?

### ISSUE THREAD

**Issue by hyandell** [libgdx/libgdx#6663]
#### Issue details
I notice the https://github.com/libgdx/libgdx/blob/master/CC-LICENSE file, which appears to be CC-BY-3.0, however I don't see what it relates to. Apologies if I'm missing something obvious - what part of the project does the CC-LICENSE apply to?

**Comment by tommyettinger** [comment]
This is a good question -- CC-LICENSE was added 9 years ago along with other licensing-related files, in this commit: https://github.com/libgdx/libgdx/tree/9b36c9525093e1afcc39a643e6cc84e02c3c19f2 . There were previously CC-LICENSE (and CC-LICENSE.txt) files in each of the sub-projects, and most of those were added about 11 years ago. The only person I know who was active then and now is @NathanSweet , since he's listed in the AUTHORS file. I know badlogic is knowledgeable about licenses and was active in that period of libGDX development, but I also know that he doesn't like getting notifications, so I'll refrain if possible. I think, but have no way of being sure, that the tests and demos are generally licensed under CC-BY-3.0, while the  non-test code is Apache 2.0. I think some specific assets may be licensed more permissively than CC-BY (such as if any are public domain, which would allow relicensing our copy to CC-BY-3.0), but none *should* be licensed incompatibly. I suspect some assets are not licensed properly, even though they should be for a CC-BY-3.0 test, like the familiar-looking https://github.com/libgdx/libgdx/blob/master/tests/gdx-tests-android/assets/data/egg.png . These should probably be replaced.

**Comment by NathanSweet** [comment]
I don't think we need the `CC-LICENSE`. Easiest to apply Apache to everything.
